prepare/envsetup.sh

```diff
@@ -406,6 +406,28 @@ local_setup_configure_nm_inf () {
     touch /tmp/inf_configured
 }
 
+install_usb_hub_driver () {
+    # Works under RHEL 8.0.
+    yum install -y libffi-devel python36-devel
+    pushd /tmp
+        wget https://acroname.com/system/files/software/brainstem_dev_kit_ubuntu_lts_18.04_no_qt_x86_64_1.tgz
+        tar xf brainstem_dev_kit_ubuntu_lts_18.04_no_qt_x86_64_1.tgz
+        cd development/python/
+        python3 -m pip install brainstem-2.7.0-py2.py3-none-any.whl; local rc=$?
```

Comment:
can we just have brainstem-2.7.0-py2.py3-none-any.whl stored in tmp directory? We do not have the latest one either (when there is a newer version we have to switch links) or is it too big?